backend/tournesol/models/entity.py

```diff
@@ -163,6 +163,32 @@ def link_to_youtube(self):
             '<a href="https://youtu.be/{}" target="_blank">Play ▶</a>', self.video_id
         )
 
+    def criteria_scores_distributions(self, poll):
+        """Returns the distribution of critera score for the entities"""
+
+        # Fetch data with QuerySet
+        contributor_rating_criteria_score_list = [
+            [y for y in x.criteria_scores.all()]
+            for x in self.contributorvideoratings.filter(poll=poll)]
+
+        contributor_rating_criteria_score_flatten_list = [
+            val for _list in contributor_rating_criteria_score_list for val in _list]
+
+        # Format data into dictionnary
+        scores_dict = dict()
+        for element in contributor_rating_criteria_score_flatten_list:
+            if element.criteria not in scores_dict:
+                scores_dict[element.criteria] = []
+            scores_dict[element.criteria].append(element.score)
```

Comment:
(minor) A [`defaultdict`](https://docs.python.org/3.9/library/collections.html#collections.defaultdict) would be convenient here:
```suggestion
        scores_dict = defaultdict(list)
        for element in contributor_rating_criteria_score_flatten_list:
            scores_dict[element.criteria].append(element.score)
```